app/javascript/components/time.svelte

```diff
@@ -0,0 +1,38 @@
+<script>
+  import { onMount } from 'svelte';
+  import { timeForTimezone } from '@lib/datetime';
+
+  export let timezone;
+  let time = new Date();
+  $: [hourMin, amPm] = timeForTimezone(timezone, time).split(' ');
+
+  onMount(() => {
+    const interval = setInterval(() => {
+      time = new Date();
+    }, 1000);
+
+    return () => {
+      clearInterval(interval);
```

Comment:
Love `svelte`'s reasonable lifecycle methods and collocated cleanup code 👍 Have you noticed if this clears up the memory leak issue?